lib/clean.js

```diff
@@ -23,7 +23,11 @@ function clean(gyp, argv, callback) {
     exists(to_delete, (found) => {
       if (found) {
         if (!gyp.opts.silent_clean) console.log('[' + package_json.name + '] Removing "%s"', to_delete);
-        return rm(to_delete, callback);
+        try {
+          fs.rmdirSync(to_delete, { recursive: true, force: true });
```

Comment:
I believe this is not correct - `rmdir` only deletes a directory if it's empty, which may not be the case here, no? https://nodejs.org/api/fs.html#fsrmdirpath-options-callback

I think it should be (also replacing the try/catch & synchronous filesystem call):
```
fs.rm(to_delete, { recursive: true, force: true }, callback);
```

Reply:
I think `rmdir` and `rm` are both correct, but that `rmdir` is deprecated, so I'll update them all. thanks!!